### PR TITLE
Add item-card specification.

### LINKED ISSUE #252: Fix item card formatting
Because we're getting empty spaces.

3 column solution: 2 for medium/large screen and 1 for small screen that has full list time order.

### PR DESCRIPTION
**Pull request by alexgerstein**
Closes #252.

### CODE REVIEW

**Comment by swglad** [comment]
This looks like it may resolve (or help to resolve) #257, but doesn't do what #252 specifies.

**Comment by swglad** [comment]
@alexgerstein Did you see the above comment?

**Comment by alexgerstein** [comment]
Ah. I'm on it.

**Comment by swglad** [comment]
Also probably goes without saying, but can you run feed-seed and make sure the feed cards look nice given the set of feeds we have loaded? If we fix our design to accommodate this set of feeds, that's not a bad thing

**Comment by alexgerstein** [comment]
How else was I going to make sure the two-col solution was working!?!

**Comment by swglad** [comment]
Talking about two separate but related issues: two vs. three columns is one, but style of feed cards vs. item cards is another. Just making sure we maximize both!

**Comment by alexgerstein** [comment]
@swglad It's ready.

File: sourcemash/frontend/templates/category.tpl [hunk 1]
```diff
@@ -7,11 +7,25 @@
 
   <div class="loading center-align"></div>
 
-  <ul id="items" class="list-group row">
-    <% items.forEach(function(item) { %>
-        <div id="item-<%= item.get('id') %>" class="item-card"></div>
-    <% }); %>
-  </ul>
+  <div id="items" class="list-group row">
+    <div class="col m6 hide-on-small-and-down">
```
Comment (swglad) [inline sourcemash/frontend/templates/category.tpl]: What's smaller than small? (not a riddle)

File: sourcemash/frontend/templates/category.tpl [hunk 2]
```diff
@@ -7,11 +7,25 @@
...
+      <% for (i = 0; i < items.length; i = i + 2) { %>
```
Comment (swglad) [inline sourcemash/frontend/templates/category.tpl]: Any reason for this type of for loop as opposed to the forEach we usually use?

Comment (swglad) [inline sourcemash/frontend/templates/category.tpl]: Ah I see how you split the two columns. I was imagining one loop with two div's inside it that alternated indices with if statements. Looks good!

File: sourcemash/frontend/static/scss/sourcemash.scss [hunk 3]
```diff
@@ -327,48 +322,50 @@ a.button-collapse.top-nav {
     }
 }
 
-.card-action {
-  @extend .grey-text;
+.item-card {
```
Comment (swglad) [inline sourcemash/frontend/static/scss/sourcemash.scss]: I noticed you didn't make any style changes on feed-cards. Everything looks good on that end (e.g. feed descriptions don't run too long and take over the card, subscribe buttons don't get pushed by long titles, etc)?

File: sourcemash/frontend/templates/category.tpl (hunk 1 above)
Comment (alexgerstein) [inline sourcemash/frontend/templates/category.tpl]: No, but might be a rap lyric...

Small is just 600px, small-and-down sets max-width of 600px.

**Comment by swglad** [comment]
Why did my merging the MixPanel branch just merge this one?

**Pull request merged**